ui-tests/cypress/support/pageObjects/pages/management/ManagementPage.js

```diff
@@ -1,5 +1,49 @@
 import {DATA_TESTID, htmlElements, WebElement} from "../../WebElement.js";
 
 import Content from "../../app/Content.js";
+import AppPage from "../../app/AppPage.js";
+import AddPage from "./AddPage.js";
 
-export default class ManagementPage extends Content {}
\ No newline at end of file
+export default class ManagementPage extends Content {
+    add = `[${DATA_TESTID}=button-step-5]`;
+    tableContainer = `${htmlElements.div}.DDTable`;
+    pageName = `[${DATA_TESTID}=common_PageTree_span]`;
+    optionsMenu = `[id=WidgetListRow-dropown]`;
+    addChild = `[aria-labelledby=WidgetListRow-dropown] li:nth-child(1) a`;
+    expandAll = `${htmlElements.div}.PageTree__toggler--expand`;
+
+    getAddButton() {
+        return this.get().find(this.add);
+    }
+
+    getTableContainer () {
+        return this.get().find(this.tableContainer);
+    }
+
+    getRowByPageName (name) {
+        const rows = this.getTableContainer().find('table tr');
+        return rows.contains(name);
```

Comment:
Shouldn't use contains to identify elements in a page object

Reply:
The idea here is finding all rows and filter the one that contains a giving string(the page name), which approach would u recommend instead of contains?

Reply:
you can (probably should) use _contains_, but since this is a validation and not a structural information it should be part of the test (_spec_ files) instead of being used in a page object

Reply:
I'm also using it as a selector for other elements, for example, on line 28, I have `getOptionsMenu` that gets the option icon element of a given row which is selected by the page name. Does it make sense?